bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -149,7 +155,7 @@ public void visit(TooFewReviewersIssue e) {
     @Override
     public void visit(InvalidReviewersIssue e) {
         var invalid = String.join(", ", e.invalid());
-        throw new IllegalStateException("Invalid reviewers " + invalid);
+        throw new BadCommitMessageException("Invalid reviewers " + invalid);
```

Comment:
I didn't mean for this new exception to be used everywhere. We were specifically trying to avoid just issues with the commit message here.

Reply:
> I didn't mean for this new exception to be used everywhere. We were specifically trying to avoid just issues with the commit message here.

I was thinking about whether to change the exception type here. In my opinion, if one of the reviewers is invalid, it is meaningless for the bot to retry, we also should make the bot end.

Reply:
Hm, you are probably right there, it was just that the specific exception name didn't fit here. But, if we report failures in the normal way instead, it's all good.